openslides/users/apps.py

```diff
@@ -20,6 +21,8 @@ def ready(self):
         from .signals import create_builtin_groups_and_admin, get_permission_change_data
         from .views import GroupViewSet, PersonalNoteViewSet, UserViewSet
 
+        from django.contrib.auth.signals import user_logged_in
```

Comment:
Is it possible to use a global import for this?

Reply:
Done.